content/blog/2018-04-22-essurvey.md

```diff
@@ -0,0 +1,337 @@
+---
+slug: "essurvey"
+title: Download data from the European Social Survey
+package_version: 1.0.0
+authors:
+  - name: Jorge Cimentada
+    url: https://cimentadaj.github.io/
+date: 2018-01-05
+categories: blog
+topicid: 925
+output: md_document
+tags:
+- r
+- community
+- data
+- review
+- onboarding
+- package
+- data-extraction
+- social-science
+---
+
+
+<!-- # see for blogpost https://github.com/ropensci/roweb2#contributing-a-blog-post -->
+Introduction

```

Comment:
Reading this intro gave me a huge smile - your embracing data science and positive community experiences. I hope the community continues to live up to this.